lib/modules/manager/pip-compile/extract.spec.ts

```diff
@@ -27,13 +27,22 @@ describe('modules/manager/pip-compile/extract', () => {
       expect(packageFile?.deps[0]).toHaveProperty('depName', 'attrs');
     });
 
+    it('returns object for setup.py', () => {
+      const packageFile = extractPackageFile(
+        Fixtures.get('setup.py'),
```

Comment:
```suggestion
        Fixtures.get('setup.py', '../pip_setup'),
```

read test fixture from `pip_setup` manager 😉